tests/utils.py

```diff
@@ -0,0 +1,32 @@
+import re
+from collections import defaultdict
+
+
+def compare_word_counts(actual: str, expected: str):
```

Comment:
The order isn't compared?

Reply:
nah, in the future we can do a stronger vector based approach if we need it but I think comparing lexicons is strong enough for now